dashbuilder-webapp/pom.xml

```diff
@@ -336,6 +336,11 @@
       <groupId>org.slf4j</groupId>
       <artifactId>log4j-over-slf4j</artifactId>
     </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
This enables Errai logging during GWT compile. However, we don't want logging impl bundled in the WAR itself, so it has `<provided>` scope.